docs/configuration/radio/device.mdx

```diff
@@ -11,24 +11,34 @@ The device config options are: Role, Serial Output, and Debug Log. Device config
+| LOST_AND_FOUND | Broadcasts location to default channel regularly for recovery efforts.                                                   | Used for recovery efforts of a lost device.                                                                                            |
+| SENSOR         | Broadcasts priority telemetry packets, optimized for sensor data collection.                                             | Deploying in scenarios where gathering environmental or other sensor data is crucial, with efficient power usage and frequent updates. |
+| TAK            | Optimized for ATAK system communication, reduces routine broadcasts.                                                     | Integration with ATAK systems for communication in tactical or coordinated operations.                                                 |
+| REPEATER       | Dedicated device that focuses solely on extending mesh network coverage by relaying messages. Not visible in Nodes list. | Best positioned in strategic locations to maximize the network's overall coverage not shown in topology.                               |
+| ROUTER         | Dedicated device that focuses solely on extending mesh network coverage by relaying messages. Visible in Nodes list.     | Best positioned in strategic locations to maximize the network's overall coverage shown in topology.                                   |
+| ROUTER_CLIENT  | Combination of both ROUTER and CLIENT.                                                                                   | Devices in a strategic position for priority routing that need to also serve as a standard CLIENT.                                     |                            
+
+#### Role Comparison
+| Device Role    | Bluetooth/Wi-Fi Enabled | Screen Enabled | Power Consumption | Retransmit | Prioritized Routing | Visible in Nodes List |
+| -------------- | ----------------------- | -------------- | ----------------- | ---------- | ------------------- | --------------------- |
+| CLIENT         | Yes                     | Yes            | Regular           | Yes        | No                  | Yes                   |
+| CLIENT_MUTE    | Yes                     | Yes            | Lowest            | No         | No                  | Yes                   |
+| CLIENT_HIDDEN  | Yes                     | Yes            | Lowest            | Known only | No                  | Yes                   |
+| TRACKER        | Optional                | No             | Low               | No         | No                  | Yes                   |
+| LOST_AND_FOUND | Optional                | No             | Regular           | No         | No                  | Yes                   |
+| SENSOR         | Optional                | No             | Lowest            | No         | No                  | Yes                   |
```

Comment:
SENSOR would use more power as it broadcasts telemetry more often ? (unless is_power_saving = true ?)

Reply:
Setting as "High"

Reply:
This one is tricky since I believe it's designed to be low power but only when power saving is enabled.

Reply:
For the purposes of this table, we are describing the initial state of the configuration. I think either Regular/High is correct.